CHANGES.rst

```diff
@@ -6,6 +6,8 @@ Changes by Version
 
 - **BREAKING** - removed ``tchannel.thrift.client_for`` and
   ``tchannel.sync.thrift.client_for`` as planned in 0.18.
+- **BREAKING** - removed ``tchannel.thrift_request_builder`` as
+  planned in 0.18.
```

Comment:
I considered that, but are not going to because of how the changelog reads between 0.18 to 0.19 - I'll create a task so we do this before landing 0.19